edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -234,21 +236,22 @@ def GetEdk2RelativePathFromAbsolutePath(self, abspath: str):
         self.logger.error(f'AbsolutePath: {abspath}')
         return None
 
-    def GetAbsolutePathOnThisSystemFromEdk2RelativePath(self, relpath: str, log_errors: Optional[bool]=True):
-        """Given a edk2 relative path return an absolute path to the file in this workspace.
+    def GetAbsolutePathOnThisSystemFromEdk2RelativePath(self, *relpath: str, log_errors: Optional[bool]=True) -> str:
+        """Given a relative path return an absolute path to the file in this workspace.
 
         Args:
-            relpath: Relative path to convert. Supports both Windows and Posix style paths.
+            *relpath: Relative path to convert. Can be the entire path or parts of the path provided separately
```

Comment:
Similar example here for consistency in documentation.

Reply:
Added the following example and verified it displayed properly in the docs:

```python
rel_path = edk2path.GetEdk2RelativePathFromAbsolutePath("C:/Workspace/edk2/MdePkg/Include")
rel_path = edk2path.GetEdk2RelativePathFromAbsolutePath("C:/Workspace", "edk2", "MdePkg", "Include")
```